docs/getting-started/server/guide.mdx

```diff
@@ -18,11 +18,13 @@ Before you start: make sure you’ve installed the recommended
 [Tools and Libraries](../tools/index.md), including:
 
 - Docker Desktop
-- Visual Studio
+- Visual Studio **
 - Powershell
 - [NET 6.0 SDK](https://dotnet.microsoft.com/download)
 - Azure Data Studio
 
+** If developing in a Windows environment, Visual Studio 2022 is required in order to compile with NET 6.0 SDK framework.  See [this article](https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/6.0/vs-msbuild-version) for more details.
+
```

Comment:
This isn't a Windows-specific requirement - Visual Studio 2022 is also available for Mac, so I assume it's required for .NET 6 in that environment as well. Let's keep it simple, please delete this note and just update the dot-point to specify `Visual Studio 2022`.

Reply:
Thanks Thomas.  I was a Windows developer by trade (retired now), so should have realized the requirements for Mac would be the same.  I updated the server guide to your suggested changes.